core/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
+name = "mc-core"
+version = "2.0.0"
+authors = ["MobileCoin"]
+edition = "2021"
+description = "MobileCoin Core Library"
+readme = "README.md"
+
+[features]
+bip39 = ["tiny-bip39", "slip10_ed25519"]
+internals = [ ]
+protos = ["prost", "prost-build"]
+serde = ["dep:serde", "mc-core-types/serde"]
+
+default = ["bip39", "heapless/default"]
+
+[dependencies]
+base64 = { version = "0.13.0", default-features = false }
+curve25519-dalek = { version = "4.0.0-pre.2", default-features = false }
```

Comment:
you mean the feature flags for dalek backends? these create a Big Problem when you're trying to cross compile so should only be enabled at the very top level to avoid multiple backends being enabled when you don't intend it...